File: Iteration1/SourceCode/src/main/java/CourseGrade.java
public class CourseGrade {
    private Course course;
    private String letterGrade;
    public CourseGrade(Course course,String letterGrade){
        this.setCourse(course);
        this.setLetterGrade(letterGrade);
    }

    public Course getCourse() {
        return course;
    }

    public void setCourse(Course course) {
        this.course = course;
    }

    public String getLetterGrade() {
        return letterGrade;
    }

    public void setLetterGrade(String letterGrade) {
        this.letterGrade = letterGrade;
    }
}

File: Iteration3/SourceCode/CourseGrade.py
class CourseGrade(object):
    def __init__(self, course, letterGrade):

        self.setCourse(course)
        self.setLetterGrade(letterGrade)

    def getCourse(self):
        return self.__course

    def setCourse(self, course):
        self.__course = course

    def getLetterGrade(self):
        return self.__letterGrade

    def setLetterGrade(self, letterGrade):
        self.__letterGrade = letterGrade